stardog/connection.py

```diff
@@ -437,7 +436,7 @@ def ask(self, query, **kwargs):
           >>> conn.ask('ask {:subj :pred :obj}', reasoning=True)
         """
         r = self.__query(query, "query", content_types.BOOLEAN, **kwargs)
-        return bool(distutils.util.strtobool(r.decode()))
+        return r.decode().lower() == "true"
```

Comment:
According to the [documentation](https://docs.python.org/3.9/distutils/apiref.html#distutils.util.strtobool) for `distutils.util.strtobool(val)`:

- True values are y, yes, t, true, on and 1; false values are n, no, f, false, off and 0. Raises [ValueError](https://docs.python.org/3.9/library/exceptions.html#ValueError) if val is anything else.

Is checking for "true" sufficient? Given that we're calling a Stardog API, this may be sufficient, but I thought I would ask, just in case we need to implement a more general replacement for `strtobool`.

Reply:
I thought about it too, but the stardog api doesn't really specify what return values these operation have, other than boolean type, take an example: https://stardog-union.github.io/http-docs/#tag/Reasoning/operation/isConsistent.
Sounds to me like it's safe to assume that we should be expecting either a True/true or False/false value.
I am going to refactor this, because I do find value with raising a ValueError when a value is something not defined.